File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BasePartyType.cs
using System.Collections.Generic;

namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BasePartyType
    {
        public string AccountExecutive { get; set; } = string.Empty;
        public string AccountExecutiveCode { get; set; } = string.Empty;
        public System.Collections.Generic.List<BaseAddressWithContactsType> Addresses { get; set; }
        public string BranchCode { get; set; } = string.Empty;
        public System.Collections.Generic.List<BaseContactType> Contacts { get; set; }
        public string Currency { get; set; } = string.Empty;
        public bool DomiciledForTax { get; set; }
        public bool DomiciledForTaxSpecified { get; set; }
        public string FileCode { get; set; } = string.Empty;
        public string SubBranchCode { get; set; } = string.Empty;
        public string TPIntroducer { get; set; } = string.Empty;
        public string TPUserCode { get; set; } = string.Empty;
        public bool TaxExempt { get; set; }
        public bool TaxExemptSpecified { get; set; }
        public string TaxNumber { get; set; } = string.Empty;
        public decimal TaxPercentage { get; set; }
        public bool TaxPercentageSpecified { get; set; }
        public string XMLDataset { get; set; } = string.Empty;
        public string Agent { get; set; } = string.Empty;
    }
}

File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BasePartyType.vb
Public Class BasePartyType
    Public Property AccountExecutive As String
    Public Property AccountExecutiveCode As String
    Public Property Addresses As List(Of BaseAddressType)
    Public Property BranchCode As String
    Public Property Contacts As List(Of BaseContactType)
    Public Property Currency As String
    Public Property DomiciledForTax As Boolean
    Public Property FileCode As String
    Public Property SubBranchCode As String
    Public Property TPIntroducer As String
    Public Property TPUserCode As String
    Public Property TaxExempt As Boolean
    Public Property TaxNumber As String
    Public Property TaxPercentage As Decimal
    Public Property XMLDataset As String
    Public Property Agent As String
    Public Property BlacklistReasonCode As String
    Public Property RenewalStopCode As String
End Class